components/ws-manager/pkg/manager/testdata/status_prebuildFail_PENDING00.golden

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
How does this test prebuild failure? I don't see any failure state

Reply:
Right, those tests are superfluous. I removed all but `STOPPED` and `STOPPING`, as those are important for the failure process. Also, formatted. :upside_down_face: